.github/workflows/build.yml

```
@@ -18,13 +18,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4, 8.0, 8.1, 8.2, 8.3, 8.4]
+        php: [8.0, 8.1, 8.2, 8.3, 8.4]
```

Should also drop 8.0
```suggestion
        php: [8.1, 8.2, 8.3, 8.4]
```